[2024-01-07 20:01:11,674] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_inference_pipeline.checking_model_prediction_ratio scheduled__2024-01-07T19:00:00+00:00 [queued]>
[2024-01-07 20:01:11,689] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_inference_pipeline.checking_model_prediction_ratio scheduled__2024-01-07T19:00:00+00:00 [queued]>
[2024-01-07 20:01:11,689] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-01-07 20:01:11,689] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-01-07 20:01:11,689] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-01-07 20:01:11,715] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): checking_model_prediction_ratio> on 2024-01-07 19:00:00+00:00
[2024-01-07 20:01:11,721] {standard_task_runner.py:52} INFO - Started process 14786 to run task
[2024-01-07 20:01:11,723] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_scoring_inference_pipeline', 'checking_model_prediction_ratio', 'scheduled__2024-01-07T19:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/lead_scoring_inference_pipeline/.ipynb_checkpoints/lead_scoring_inference_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmp08hsdfof', '--error-file', '/tmp/tmpm10luoko']
[2024-01-07 20:01:11,724] {standard_task_runner.py:80} INFO - Job 54: Subtask checking_model_prediction_ratio
[2024-01-07 20:01:11,816] {task_command.py:371} INFO - Running <TaskInstance: Lead_scoring_inference_pipeline.checking_model_prediction_ratio scheduled__2024-01-07T19:00:00+00:00 [running]> on host a3830b6d5f93
[2024-01-07 20:01:12,023] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_scoring_inference_pipeline
AIRFLOW_CTX_TASK_ID=checking_model_prediction_ratio
AIRFLOW_CTX_EXECUTION_DATE=2024-01-07T19:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-07T19:00:00+00:00
[2024-01-07 20:01:12,029] {logging_mixin.py:115} INFO - Error while running prediction_ratio_check : Execution failed on sql 'select * from predictions': no such table: predictions
[2024-01-07 20:01:12,029] {python.py:173} INFO - Done. Returned value was: None
[2024-01-07 20:01:12,046] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_scoring_inference_pipeline, task_id=checking_model_prediction_ratio, execution_date=20240107T190000, start_date=20240107T200111, end_date=20240107T200112
[2024-01-07 20:01:12,096] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-07 20:01:12,166] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
